static/js/jira-configuration.js

```diff
@@ -23,6 +23,17 @@ $(".add-organization-link").click(function(event) {
 	});
 });
 
+$(".select-github-version-link").click(function(event) {
+	event.preventDefault();
+
+	AP.navigator.go(
+		'addonmodule',
+			{
+				moduleKey: "github-select-version-page"
```

Comment:
This will eventually need to be dynamic as we will be able to hit this fro 2 different points in the flow: 1. from the config page when state is empty or 2. from the connect an org page when state is not empty. However, I'm not close to that point so hardcoded works for this PR.